src/auro-calendar-cell.js

```diff
@@ -307,7 +323,7 @@ export class AuroCalendarCell extends LitElement {
     }
 
     this.calendarMonth = this.runtimeUtils.closestElement('auro-calendar-month', this);
-    this.auroPopover = this.shadowRoot.querySelector('auro-popover');
+    this.auroPopover = this.shadowRoot.querySelector(this.popoverTag._$litStatic$);
```

Comment:
Why not use the change line 303 above to be this.auroPopover? No need to have a const and a this.auroPopover set to the same value. I know it was that way before, but let's take the opportunity to clean up the performance of the component there.

Reply:
Fixed.